package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "sls-financial-data-api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "tsc",
+    "deploy": "npm run build && sls deploy"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@types/aws-lambda": "^8.10.31",
```

Comment:
types should be in devdependencies. Install with npm install -D {package}. Types don't end up in the final package so they are dev only